Formula/pastebinit.rb

```diff
@@ -20,7 +19,7 @@ def install
     inreplace "pastebinit", "/usr/local/etc/pastebin.d", etc + "pastebin.d"
     system "docbook2man", "pastebinit.xml"
     bin.install "pastebinit"
-    etc.install "pastebin.d"
+    cp_r "pastebin.d", etc
```

Comment:
In this release they seem to have updated pastebin's `api_dev_key` in a configuration file, `/usr/local/etc/pastebin.d/pastebin.com.conf` and the client works out of the box without configuring a personal api token. When upgrading this package, `etc.install` doesn't copy, if configuration files already exist.